app/forms/participants/participant_validation_form.rb

```diff
@@ -45,6 +45,10 @@ def attributes
     def date_of_birth=(value)
       @date_of_birth_invalid = false
       @date_of_birth = ActiveRecord::Type::Date.new.cast(value)
+
+      if invalid_date?(value)
+        @date_of_birth_invalid = true
+      end
```

Comment:
Do we want to move this before the assignment and add an early return?

Reply:
Returning early is causing problems with some tests, notably feature tests... doing `@date_of_birth = value` in the check, makes it blank but with the invalid date message.